src/common/telemetry.ts

```diff
@@ -0,0 +1,293 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
We don't provide `AbstractTelemetryService` because `telemetryService` in extension need to extend `Disposable`